command.php

```diff
@@ -203,6 +203,10 @@ private static function pass_through_pager( $out ) {
 			return 0;
 		}
 
+		if ( false === ( $pager = getenv( 'PAGER' ) ) ) {
+			$pager = 'less -r';
```

Comment:
Happy to add that if you guys want. I wasn't sure since Windows is explicitly skipped above on L200-204. I have no way to test it though :smile: